rclcpp/include/rclcpp/client.hpp

```diff
@@ -42,32 +46,28 @@ class ClientBase
 
   RCLCPP_PUBLIC
   ClientBase(
-    std::shared_ptr<rmw_node_t> node_handle,
-    rmw_client_t * client_handle,
+    std::shared_ptr<rcl_node_t> node_handle,
     const std::string & service_name);
 
   RCLCPP_PUBLIC
-  virtual ~ClientBase();
```

Comment:
Why did you remove the destructor declaration? I think it might be important that it is declared `virtual`.